pages/profile/read/views/profile.jsx

```diff
@@ -23,7 +21,7 @@ class Profile extends React.Component {
     } = this.props.profile;
 
     const allowedActions = this.props.allowedActions;
-    const title = this.props.title;
+    const estName = this.props.estName;
```

Comment:
Does changing this prop mean it's going to be a breaking change on the dashboard, and if so should we revert to `title`?

Reply:
It is not breaking the dashboard, because I already changed it there - my thinking is, this is not a title, but the establishment name - not sure, but I will rename it

Reply:
👍